app/src/components/main/analytics/AnalyticsWrapper.jsx

```diff
@@ -63,7 +66,12 @@ export class AnalyticsWrapper extends Component {
 
     ReactGA.initialize(GOOGLE_ANALYTICS_INSTANCE);
     ReactGA.pageview(window.location.pathname + window.location.search);
-    ReactGA.set({ dimension1: instanceId, dimension2: appVersion });
+    ReactGA.set({
+      dimension1: instanceId,
+      dimension2: appVersion,
+      dimension3: userId,
+      dimension4: Date.now(),
```

Comment:
Could you please explain why you need this parameter?

Reply:
to add user data to each event

Reply:
I meant `dimension4: Date.now()`